### ISSUE TITLE
Variables on appropriate place

### ISSUE THREAD

**Issue by PerPalmgren** [jasp-stats/jasp-issues#1627]
### Is your feature request related to a problem?

In statistics it is common practice to insert the dependent variable on the Y axis and the independent variable on the X- axis. Of course, in correlations it does not matter but in regressions it does. However, often the variable inserted at the TOP in a correlation analysis is the dependent variable. Thus, it would be good with coherency when running JASP. Right now, when running a scatter plot for the Descriptive modules when I insert the dependent variable on the TOP (see picture) it ends up in the results on the X-ais (see picture). This makes my students slightly confused For clarification of pictures n Swedish “Hoppjöjd” means jump height and is = dependent variable while “Benstyrka” is leg strength and is = independent variable.

### Describe the solution you would like

I would suggest that the scatter plot function in the Descriptive module would work on the same premises as in the Regression module (more specifically correlation), thus when running a scatter plot the variable inserted at the TOP (see picture) ends up on the Y-axis (picture).

### Additional context

![Picture1](https://user-images.githubusercontent.com/79397662/154836642-70ed29be-578c-4c86-b8d9-e4dfbbb34f63.png)
![Picture2](https://user-images.githubusercontent.com/79397662/154836645-2de12ff2-b7dd-42a8-9fe0-fd7acde28203.png)
![Picture3](https://user-images.githubusercontent.com/79397662/154836650-f8c62acc-fe38-46c3-8260-505b53f8c415.png)
![Picture4](https://user-images.githubusercontent.com/79397662/154836653-d29f7c0a-96cc-4d34-abd6-d2ec92fb6f94.png)
![Picture4](https://user-images.githubusercontent.com/79397662/154836654-30e56388-fb7a-48d5-bbf5-c1f49a86bc91.png)

**Comment by PerPalmgren** [comment]
Hi, 
Will this be possible to rectify and implement for the 0.17 version? I guess it is not a substantial change.
All the best
Per

**Comment by Kucharssim** [comment]
Yes, I will take a look into it by then!

**Comment by PerPalmgren** [comment]
Hi,
Is this feature incorporated now in the new version?

**Comment by juliuspf** [comment]
Simon knows this better than me, but since the project it is assigned to is 0.17, I doubt it is in 0.16.2, nor will it be in 0.16.3. I apologize for the delay.

**Comment by PerPalmgren** [comment]
So, it is sure it will not be in 0.16.3?😢

**Comment by PerPalmgren** [comment]
It seems like in JASP 0.16.3 this function has also disappeared  in the correlation module😥. I really think it would be beneficial if in ALL scatter plots the inserted top variable  always ends up on the Y-axis (so the dependent variable gets accurately placed).
Kind reagards
Per

**Comment by Kucharssim** [comment]
> It seems like in JASP 0.16.3 this function has also disappeared in the correlation module😥

Can you clarify? We certainly did not remove any functionality...

**Comment by PerPalmgren** [comment]
Sorry I was wrong. It seems to work now. In the Regression module (more specifically correlation), thus when running a scatter plot the variable inserted at the TOP (see picture) ends up on the Y-axis. But as I stated above it would be nice to have this feature for the scatter plots in the descriptive as well.

**Comment by Kucharssim** [comment]
Yes, I am looking into it right now, but I am not sure whether we already fixed it, or whether I don't understand the issue... It seems that the two analyses show the same variables on the x and y axis:

1. Descriptives

![Screenshot 2022-06-22 at 12 09 56](https://user-images.githubusercontent.com/7093546/175004434-82886166-0d3b-4bb4-92dd-60abb4e07ff8.png)

2. Correlation

![Screenshot 2022-06-22 at 12 10 53](https://user-images.githubusercontent.com/7093546/175004533-13076932-c625-4aaa-b9ff-b2a35ad2c179.png)


Now I think we could make a better job at making sure that the two plots looks *identical*, but I don't see an issue with the flipped axes... Or is there some setting I am missing to select? Thanks!

**Comment by PerPalmgren** [comment]
It is not the case when I run it under customizable plots (scatter plots) in the descriptive module (see below).
![Namnlös](https://user-images.githubusercontent.com/79397662/175007838-9a8c13c6-d499-49ad-9a25-9b01def0f1a6.png)

**Comment by PerPalmgren** [comment]
Now it is me who do not really follow. I would argue in regressions in statistics the dependent variable is always put on the Y axis. So, I think it would be most logic to always let the variable inserted first (on the top) to end up on the y axis. Of course in simple correlations it does not matter but if the second step to correlation is a regression (very, very common) it would make sense to use the same. By many experience many researchers use the descriptive plots to investigate assumptions. Question! In you example is contGamma your outcome variable and contNormal your predictor (in a presumtive simple regression)?

**Comment by Kucharssim** [comment]
> Now it is me who do not really follow.

Sorry for the confusion!

> I would argue in regressions in statistics the dependent variable is always put on the Y axis.

Yes, we agree on this.

> So, I think it would be most logic to always let the variable inserted first (on the top) to end up on the y axis.

This is where we disagree: I don't see any reason to assume that the first variable put into the variables box in descriptives or correlations is the dependent variable. This is something people may find different preferences for.

> Question! In you example is contGamma your outcome variable and contNormal your predictor (in a presumtive simple regression)?

All screenshots I posted were made by entering `contNormal` first and `contGamma` second. So if you consider the first variable "outcome", then `contNormal` is the outcome and `contGamma` the predictor. I don't really think it's that productive to think about a "predictor" and a "outcome" in descriptives or correlations though. What I am focusing on here is that the axes should be consistent between the correlation matrix plots and the pairwise scatter plots both in descriptives and correlations. I will make sure that in this example `contNormal` would end up on the y-axis in all plots, which I think is exactly what you wanted in the first place :)

**Comment by PerPalmgren** [comment]
I still think it would be of value if all all scatter plots (both in descriptive and correlation module) used the default that the variable inserted on the top (in the variable box) ALWAYS ended up on the Y-axis in the plots.

**Comment by PerPalmgren** [comment]
What heppened with this issue?

**Comment by PerPalmgren** [comment]
I understand completely. Thank you for not letting this pass under the radar.
All the best
Per

Per J. Palmgren | DC, FRCC, MMedEd, PhD |
University Lecturer |
Department for Learning, Informatics, Management and Ethics |
Karolinska Institutet | Tomtebodavägen 18A | 171 77 Stockholm |
Tel. + 46 8 524 85 294
________________________________

Karolinska Institutet - a medical university

**Comment by PerPalmgren** [comment]
@Kucharssim 
Dear Simon,
Just wanted to check in on this?
All the best 
Per😎

**Comment by tomtomme** [comment]
@PerPalmgren 
Kucharissm is working on it. But I can give not timeline.

**Comment by tomtomme** [comment]
this is being worked on here https://github.com/jasp-stats/jaspGraphs/pull/100

**Comment by tomtomme** [comment]
@JohnnyDoorn 
Do you still plan to do the reorganization of the descriptives module in the near term?
Because the situation here got even worse with 0.95.

See renamed to zip jasp file:
[Album Sales.jasp.zip](https://github.com/user-attachments/files/21032859/Album.Sales.jasp.zip)

This is now not only inconsistent within descriptives but also within the correlation module (and between all modules that do scatterplots). The modules that do name explicitely which will be x and y axis are plot builder and flexplot. flexplot has y on top, plot builder has x on top in the gui. But this is not that bad, since you know beforehand what you get.

A way to solve this would also be to give descriptives and correlation modules also distinctive AV and UV boxes instead of the unified box. This would make sense, since even these modules draw regression lines.
The quicker solution would be to just modify the plots in the correl & descr modules and always put the top variable on the y axis as suggested by the OP.
